File: docs-old/ide/codesnippet/CSharp/walkthrough--test-first-support-with-the-generate-from-usage-feature_4.cs
    public class Automobile
    {
        public string Model { get; set; }
        public int TopSpeed { get; set; }

        public Automobile(string model, int topSpeed)
        {
            this.Model = model;
            this.TopSpeed = topSpeed;
        }

        public Automobile()
        {
            // TODO: Complete member initialization
        }

        public void Start()
        {
            throw new NotImplementedException();
        }

        public bool IsRunning { get; set; }
    }

File: docs-old-1103-2/ide/codesnippet/VisualBasic/walkthrough-test-first-support-with-the-generate-from-usage-feature_4.vb
    Public Class Automobile
        Sub New(ByVal model As String, ByVal topSpeed As Integer)
            _model = model
            _topSpeed = topSpeed
        End Sub
        Sub New()
            ' TODO: Complete member initialization 
        End Sub

        Property Model() As String
        Property TopSpeed As Integer
        Property IsRunning As Boolean
        Sub Start()
            Throw New NotImplementedException
        End Sub
    End Class